code/src/App.js

```diff
@@ -1,9 +1,113 @@
-import React from 'react'
+import React, {useState} from 'react'
+
+
+import FavoriteBook from './Components/FavoriteBook';
+import HowManyBooks from './Components/HowManyBooks';
+import FavoriteGenre from './Components/FavoriteGenre';
+import Header from './Components/Header';
+import Summary from './Components/Summary';
+
+import './index.css'
+
+
 
 export const App = () => {
+
+  //function for submit button
+  const handleSubmit = (event) => {
+
+    event.preventDefault()
+    if (isSurveyComplete())
+    {
+      setDisplaySummary(true)
+    }
+    else{
+      //if survey isn't complete, it will alert and scroll to top.
+      alert("Survey not complete");
+      window.scroll(0,0);
+    }
+  }
+
+  const [number, setNumber] = useState("");
+  const [favoriteGenre, setFavoriteGenre] = useState("")
+  const [book, setBook] = useState("");
+  const [displaySummary, setDisplaySummary] = useState(false);
+
+  const onNumberChange = (event) => {
+    setNumber(event.target.value)
+  }
+
+  const onGenreChange = (event) => {
+    setFavoriteGenre(event.target.value)
+  }
+
+  const onBookChange = (event) => {
+    setBook(event.target.value);
+  }
+
+  const isSurveyComplete =() => {
+    if (number === '') {
+      return false
+    }
+    if (favoriteGenre === ''){
+      return false
+    }
+    if (book === ''){
+      return false
+    }
+    return true
+  }
+
+
   return (
-    <div>
-      Find me in src/app.js!
+
+    <div >
+      {!displaySummary ? ( // if displaySummary is false
+
+        <div className="main-container">
+          <Header />
+
+          <div className='formContainer'>
+          <form onSubmit={handleSubmit}>
+          <section className="many" id="many">
```

Comment:
This className is a little bit vague, if the application was bigger it might be a bit hard to figure out what it stands for.